Name: Hide Mobile Hotspot
Version: 2.2.4-1
Author: webOS Archive
Description: Hide Mobile Hotspot from the US Pre3 Launcher

--- .orig/usr/palm/applications/com.palm.app.mobilehotspot/appinfo.json
+++ /usr/palm/applications/com.palm.app.mobilehotspot/appinfo.json
@@ -8,7 +8,8 @@
 	"noWindow": "true",
 	"version": "1.1.12",
 	"vendor": "HP",
-	"removable": false,
+	"removable": true,
+	"visible": false,
 	"keywords": ["Hotspot","WiFi","Mobile"]
 }
 
